File: 01_입출력과 사칙연산/박광현/10430.swift

let input = readLine()!
let inputArr = input.split(separator: " ")
let a = Int(inputArr[0])!
let b = Int(inputArr[1])!
let c = Int(inputArr[2])!
print((a+b)%c)
print(((a%c)+(b%c))%c)
print((a*b)%c)
print(((a%c)*(b%c))%c)

File: 01_입출력과 사칙연산/이정현/10430.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().split(' ').map(val=>+val);
const [a,b,c] = input

console.log( (a+b)%c );
console.log( ((a%c) + (b%c))%c );
console.log( (a*b)%c);
console.log( ((a%c) * (b%c))%c );